src/client/components/pages/parts/collections-table.js

```diff
@@ -90,7 +90,14 @@ class CollectionsTable extends React.Component {
 						{tableHeading}
 					</h1>
 					<div className="text-right">
-						{newCollectionButton}
+						{/*
+						1.Check if user is logged In if so checks the page ownerId with users id
+						OR
+						2.Check if user is logged In if so checks the page is central public collections page or not
+						*/}
+						{((user && user.id === ownerId) ||
+							(user && !ownerId)) &&
```

Comment:
It would be nice to extract this and the comments that accompany it to a new constant `showNewCollectionButton` for example, above in this same render block.

Better legibility, and no odd comments being rendered in the page html ;)

Reply:
Made requested changes in the latest commit.